examples/toolchains/rust/nixpkgs.json

```diff
@@ -2,6 +2,6 @@
   "owner": "NixOS",
   "repo": "nixpkgs",
   "branch": "22.05",
-  "rev": "ce6aa13369b667ac2542593170993504932eb836",
-  "sha256": "0d643wp3l77hv2pmg2fi7vyxn4rwy0iyr8djcw1h5x72315ck9ik"
+  "rev": "73247c5ab2bc2037d7dd0d310f1fb43ed191edad",
+  "sha256": "1213j2gvsmzmhpyi2jyn729s9d0gbn56k3d873iknd5ilb2rvpcr"
```

Comment:
What channel is that from? It doesn't look like it's 22.05 anymore. Would 22.11 suffice (as being updated [here](https://github.com/tweag/rules_nixpkgs/pull/313/files#diff-e539c550d3ff8868701fc7fc237c08a365c969ec8561a6104205c9bfd0d4e278)), or does it have to be more recent? If so, could you update the branch field accordingly?